docs/UserGuide.md

```diff
@@ -78,13 +73,100 @@ Fig 1. Home tab
 {:.caption}
 
 #### 2.2.2 Inventory & Statistics Tab
+
 Displays [inventory](#32-inventory) and [statistics](#35-statistics).
 
 ![Inventory & Statistics tab](images/product-screenshots/general/BarChart.png)
 
 Fig 2. Inventory & Statistics tab
 {:.caption}
 
+### 2.3 Creating your first order
+
+This is a step-by-step guide to creating your first order. If you have not installed BobTheBistroBoss, you
+may do so by following the instructions [here](#21-installation).
+
+For this guide, we will simulate the following scenario:
+
+Your regular customer John Doe wishes to place two orders, one on 10 December 2021 and the other on 11 December 2021.
+Both of these orders are to be delivered at 12pm. For both of these orders, he will require 3 tuna sandwiches. Each
+tuna sandwich will be priced at $2.
+
+Here are the details of John Doe:
+* Name: John Doe
+* Phone Number: 92345678
+* Email: johndoe@gmail.com
+* Address: Blk 422, Bedok North Road, #15-25
+
+Here are the ingredients that will be used for the tuna sandwich:
+
+* 2 slices of bread
+* 1 slice of cheese
+* 1 can of tuna
+
+Suppose you currently have these ingredients with you but have not been added in the inventory:
+
+* 100 slices of bread
+* 100 slices of cheese
+* 100 cans of tuna
+
+#### 2.3.1 Approach 1
+
+We can proceed to add the first order on 10 December 2021 by entering the following
+command in the command box at the bottom of the app:
+
+`add-o cn/John Doe cp/92345678 ca/Blk 422, Bedok North Road, #15-25 rn/Tuna Sandwich ri/Bread-2-slices, Cheese-1-slices, Tuna-1-cans op/6 oq/3 od/10-12-2021 1200`
+
+You will notice that the order has been added to the Orders list (orders are sorted by deadline). However, this is not
+the recommended approach, especially for repeated orders, as it can be very inefficient. Moreover, this approach does
+not allow you to track ingredients in your inventory. We will now add the second order using our recommended approach.
+
+#### 2.3.2 Approach 2 (Recommended)
+
+Firstly, switch over to the Inventory & Statistics tab by entering the following command:
+
+`tab 2`
+
+Now, we will proceed to add our 3 ingredients to the inventory by entering the following commands:
+
+`add-i in/Bread iq/100 iu/slices`
+
+`add-i in/Cheese iq/100 iu/slices`
+
+`add-i in/Tuna iq/100 iu/cans`
+
```

Comment:
All these commands should be updated because after the first order you would have less quantity